usage/app/lib/CrierEventProcessor.scala

```diff
@@ -99,42 +104,50 @@ abstract class CrierEventProcessor(config: UsageConfig, usageGroupOps: UsageGrou
       "requestId" -> UUID.randomUUID().toString
     )
 
-    val dateTime: DateTime = new DateTime(event.dateTime)
+    Try {
+      val dateTime: DateTime = new DateTime(event.dateTime)
 
-    event.eventType match {
-      case EventType.Update =>
+      event.eventType match {
+        case EventType.Update =>
 
-        event.payload match {
-          case Some(content: EventPayload.Content) =>
-            getContentItem(content.content, dateTime)
-              .emitAsUsageGroup(CrierUsageStream.observable, usageGroupOps)
-          case _ =>
-            logger.debug(logMarker, s"Received crier update for ${event.payloadId} without payload")
-        }
-      case EventType.Delete =>
+          event.payload match {
+            case Some(content: EventPayload.Content) =>
+              getContentItem(content.content, dateTime)
+                .emitAsUsageGroup(CrierUsageStream.observable, usageGroupOps)
+            case _ =>
+              logger.debug(logMarker, s"Received crier update for ${event.payloadId} without payload")
+          }
+        case EventType.Delete =>
         //TODO: how do we deal with a piece of content that has been deleted?
-      case EventType.RetrievableUpdate =>
-
-        event.payload match {
-          case Some(retrievableContent: EventPayload.RetrievableContent) =>
-            val capiUrl = retrievableContent.retrievableContent.capiUrl
-
-            val query = ItemQuery(capiUrl, Map())
-
-            liveCapi.getResponse(query).map(response => {
-
-              response.content match {
-                case Some(content) =>
-                  LiveContentItem(content, dateTime)
-                    .emitAsUsageGroup(CrierUsageStream.observable, usageGroupOps)
-                case _ =>
-                  logger.debug(logMarker, s"Received retrievable update for ${retrievableContent.retrievableContent.id} without content")
-              }
-            })
-          case _ => logger.debug(logMarker, s"Received crier update for ${event.payloadId} without payload")
-        }
-
-      case _ => logger.debug(logMarker, s"Unsupported event type $EventType")
+        case EventType.RetrievableUpdate =>
+
+          event.payload match {
+            case Some(retrievableContent: EventPayload.RetrievableContent) =>
+              val capiUrl = retrievableContent.retrievableContent.capiUrl
+
+              val query = liveCapi.usageQuery(retrievableContent.retrievableContent.id)
```

Comment:
this line is the bugfix, the rest are logging improvements and a tiny refactor to do this DRYly

Reply:
the bug being: the capi client expects to query using the capi ID, but we were giving it the capi URL